exercise7/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml.Linq;
+using Newtonsoft.Json;
+using Retrofit.Net;
+
+namespace exercise7
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string[] urls =
+            {
+                "http://danabus.vn/data/2/bus-services.json",
+                "http://danabus.vn/data/2/bus-services/5.json",
+                "http://danabus.vn/data/2/bus-services/7.json",
+                "http://danabus.vn/data/2/bus-services/8.json",
+                "http://danabus.vn/data/2/bus-services/11.json",
+                "http://danabus.vn/data/2/bus-services/12.json"
+            };
+            foreach (var url in urls)
+            {
+                ConvertJsonToXml(url);
```

Comment:
Declare `urls` as a list then you can call `urls.ForEach(ConvertJsonToXml)`

Reply:
done